src/utilities/util_verify.c

```diff
@@ -75,21 +80,21 @@ util_verify(WT_SESSION *session, int argc, char *argv[])
         return (1);
 
     if (dump_address || dump_blocks || dump_layout || dump_offsets != NULL || dump_pages ||
-      stable_timestamp) {
-        size = strlen("dump_address,") + strlen("dump_blocks,") + strlen("dump_layout,") +
-          strlen("dump_pages,") + strlen("dump_offsets[],") +
+      stable_timestamp || dump_data) {
+        size = strlen("dump_data") + strlen("dump_address,") + strlen("dump_blocks,") +
```

Comment:
One overall comment, `dump_user_data`, or `dump_app_data` might be clearer than `dump_data`, but you've thought about this harder than I have, I'd defer to you on this.

Reply:
Changed it to `dump_app_data`